bigquery/snippets/snippet.go

```diff
@@ -641,6 +632,7 @@ func copyTable(client *bigquery.Client, datasetID, srcID, dstID string) error {
 	return nil
 }
 
+// Create a quick table by issuing a CREATE TABLE AS SELECT query.
```

Comment:
Thanks for adding this comment. Would you mind changing it to start with `generateTableCTAS`?

"Comments should begin with the name of the thing being described," from https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences.

Reply:
Done.